modules/java/examples/code_snippets/Examples.java

```diff
@@ -748,26 +748,44 @@ public void testDatabaseEncryption() throws CouchbaseLiteException {
     // ### Logging
     public void testLogging() {
         // tag::logging[]
+        // deprecated
         Database.setLogLevel(LogDomain.REPLICATOR, LogLevel.VERBOSE);
         Database.setLogLevel(LogDomain.QUERY, LogLevel.VERBOSE);
+
         // end::logging[]
     }
 
     public void testEnableCustomLogging() {
         // tag::set-custom-logging[]
         // this custom logger will never be asked to log an event
         // with a log level < WARNING
-        Database.log.setCustom(new LogTestLogger(LogLevel.WARNING));
+        Database.log.setCustom(new LogTestLogger(LogLevel.WARNING)); // <.>
         // end::set-custom-logging[]
     }
 
 
+    // ### Console logging
+    public void testConsoleLogging() throws CouchbaseLiteException {
+      // tag::console-logging[]
+          Database.log.getConsole().setDomain(LogDomain.ALL_DOMAINS); // <.>
+          Database.log.getConsole().setLevel(LogLevel.VERBOSE); // <.>
+
+      // end::console-logging[]
+    }
+
+
     // ### File logging
     public void testFileLogging() throws CouchbaseLiteException {
        // tag::file-logging[]
-        final File path = new File("/usr/local/MyApp/logs")
-        Database.log.getFile().setConfig(new LogFileConfiguration(path.toString()));
-        Database.log.getFile().setLevel(LogLevel.INFO);
+          final File path = new File("/usr/local/MyApp/logs")
```

Comment:
I suggest, instead of "/usr/local", you use System.getProperty("user.dir")).  Nothing really wrong with this, as is.  On most systems, though, it probably won't actually work...